packages/@okta/vuepress-site/docs/reference/api/system-log/index.md

```diff
@@ -280,38 +280,38 @@ In the Okta event data object, the UserAgent object provides specifications abou
+Geographical context describes a set of geographic coordinates. In addition to containing latitude and longitude data, the GeographicalContext object also contains address data of postal code-level granularity. Within the [Client](#client-object) object, the geographical context refers to the physical location of the client when it sends the request that triggers this event. All [Transaction](#transaction-object) events with `type` equal to `WEB` has a geographical context set. [Transaction](#transaction-object) events with `type` equal to `JOB` doesn't have a geographical context set. The geographical context data can be missing if the geographical data for a request can't be resolved.
 
 | Property    | Description                                                                                                          | DataType                                  | Nullable |
 | ----------- | ----------------------------------------------------------------------------------                                   | -------------------------------------     | -------- |
 | geolocation | Contains the geolocation coordinates (latitude, longitude)                                                           | [Geolocation object](#geolocation-object) | TRUE     |
-| city        | The city encompassing the area containing the geolocation coordinates, if available (e.g. Seattle, San Francisco)    | String                                    | TRUE     |
-| state       | Full name of the state/province encompassing the area containing the geolocation coordinates (e.g. Montana, Incheon) | String                                    | TRUE     |
-| country     | Full name of the country encompassing the area containing the geolocation coordinates (e.g. France, Uganda)          | String                                    | TRUE     |
-| postalCode  | Postal code of the area encompassing the geolocation coordinates                                                     | String                                    | TRUE     |
+| city        | The city that encompasses the area that contains the geolocation coordinates, if available (for example, Seattle, San Francisco)    | String                                    | TRUE     |
+| state       | Full name of the state/province that encompasses the area that contains the geolocation coordinates (for example, Montana, Incheon) | String                                    | TRUE     |
```

Comment:
as per MMS, remove the slash (or) -- and I think Incheon is more known as a city not a province or state. But I'm not up on my South Korean geography.

Reply:
Incheon is definitely not a province... and we don't need two examples anyway!